### DISCUSSION TITLE
Didn't get the student benifits

### DISCUSSION THREAD

**Discussion by johns-del** [community/community#135776] [GitHub Education]
### Hi everyone,

I have been approved in Aug 11 but I do not receive my benefit

**Comment by davevad93** [comment]
Hi @johns-del ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @johns-del was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by prabhjotschugh** [comment]
Hi
After approval, it can take about 5-7 days to receive the benefits. You will get an email regarding the same.
Hope it helps!
